src/app/view/perspectives/gantt-chart/tasks/visualization/gantt-chart-visualization.component.ts

```diff
@@ -45,13 +45,19 @@ export class GanttChartVisualizationComponent implements OnChanges {
   public currentMode: GanttChartMode;
 
   @Output()
-  public patchData = new EventEmitter<{
+  public datesChange = new EventEmitter<{
     dataResourceId: string;
-    type: AttributesResourceType;
-    collectionConfigId: string;
+    resourceType: AttributesResourceType;
     changes: {attributeId: string; value: any}[];
   }>();
 
+  @Output()
+  public progressChange = new EventEmitter<{
+    dataResourceId: string;
+    resourceType: AttributesResourceType;
+    change: {attributeId: string; value: any};
```

Comment:
I can see this data structure defined multiple times. Maybe it would be better to extract it as an interface.